.eslintrc.js

```diff
@@ -2,17 +2,16 @@ module.exports = {
   env: {
     browser: true,
     es6: true,
+    jest: true
   },
   extends: 'airbnb-base',
   globals: {
     expect: 0,
```

Comment:
please also remove other global variables for test environment such as `expect`, `describe`, `beforeEach`, `afterEach`